modules/cactus-web/src/Accordion/Accordion.tsx

```diff
@@ -208,6 +288,39 @@ export const AccordionProvider = (props: AccordionProviderProps) => {
     setState(newState)
   }
 
+  const manageFocus = (id: string, offset: number) => {
+    if (managedAccordions.current.length > 1) {
+      const currentIndex = managedAccordions.current.findIndex(accordionId => accordionId === id)
+      let nextIndex = currentIndex + offset
+      if (nextIndex < 0) {
+        nextIndex = managedAccordions.current.length - 1
+      } else if (nextIndex >= managedAccordions.current.length) {
+        nextIndex = 0
+      }
+      focusById(managedAccordions.current[nextIndex])
+    }
+  }
+
+  const focusFirst = () => {
+    if (managedAccordions.current.length > 1) {
+      focusById(managedAccordions.current[0])
+    }
+  }
+
+  const focusLast = () => {
+    if (managedAccordions.current.length > 1) {
+      focusById(managedAccordions.current[managedAccordions.current.length - 1])
+    }
+  }
+
+  const focusById = (id: string) => {
+    const focusAccordion = document.querySelector(`[data-id="${id}"]`)
```

Comment:
```suggestion
    const focusAccordion = document.querySelector(`#${id}`)
```
You should be able to query id directly with just #, or was there a different purpose for adding it also as a data-* attribute?